### PR TITLE
[MRG] add whats new file

### PR DESCRIPTION
**Pull request by jasmainak**
cc @sappelhoff @teonbrooks @monkeyman192 @choldgraf 

let me know if I forgot any important contributions ...

### CODE REVIEW

**Comment by sappelhoff** [comment]
PR #55 included quite a lot of stuff, I would add:

```
# BUG
- Fixed `_check_types` and added tests for `_read_raw`, `_check_types` `_parse_ext` by `Stefan Appelhoff`_ `#55 <https://github.com/mne-tools/mne-bids/pull/55>`_
- Save neural data of all ALLOWED_EXTENSIONS, not only fif by `Stefan Appelhoff`_ `#55 <https://github.com/mne-tools/mne-bids/pull/55>`_

# API
- Restructure `meg_bids.py` to be more permissive for other modalities and rename to `mne_bids.py` by `Stefan Appelhoff`_ `#55 <https://github.com/mne-tools/mne-bids/pull/55>`_

```

**Comment by jasmainak** [comment]
@sappelhoff we only link to public API in `whats_new.rst`. It's more user facing. I've skipped all the pull requests related to maintenance for instance. Can you rephrase that in more user-friendly terms?

**Comment by sappelhoff** [comment]
ahh I see :-)

How about this?

```
# BUG
- The source neural data is now copied over to the new BIDS directory. Previously this was only true for FIF data. By `Stefan Appelhoff`_ `#55 <https://github.com/mne-tools/mne-bids/pull/55>`_

# API
- Restructuring of package: `meg_bids.py`  was renamed to `mne_bids.py` and is more open towards the integration of other modalities (iEEG, EEG) by `Stefan Appelhoff`_ `#55 <https://github.com/mne-tools/mne-bids/pull/55>`_

```

**Comment by jasmainak** [comment]
@sappelhoff included this after slight rephrasing. And we don't have the `API` heading now because we haven't released yet. So, I squashed it into the `Changelog` part. 

Merge if you're happy!

**Comment by sappelhoff** [comment]
Thanks @jasmainak :-)
